src/dshould/contain.d

```diff
@@ -0,0 +1,114 @@
+module dshould.contain;
+
+import dshould.ShouldType;
+
+unittest
+{
+    import dshould.basic : not, should;
+
+    [2, 3, 4].should.contain([3]);
+    [2, 3, 4].should.contain([4, 3]);
```

Comment:
This is misleading: the sequence [2, 3, 4] does not contain a sub-sequence [4, 3].
Fluent assertions has `containAll` and `containAny` in addition.